ldk/node/package.json

```diff
@@ -72,6 +72,7 @@
     "grpc_tools_node_protoc_ts": "4.1.4",
     "husky": "4.3.0",
     "jest": "26.4.2",
+    "jest-create-mock-instance": "1.1.0",
```

Comment:
I'm not a fan of adding this, but I was struggling pretty hard to get jest and ts-jest to do this for me without complaint.